OpenRA.Mods.Common/Traits/Parachutable.cs

```diff
@@ -71,7 +72,8 @@ public void OnLanded()
 			if (sequence != null && palette != null)
 				self.World.AddFrameEndTask(w => w.Add(new Explosion(w, self.OccupiesSpace.CenterPosition, sequence, palette)));
 
-			self.Kill(self);
+			if (self.World.ActorMap.GetActorsAt(self.Location).Any(a => a != ignore))
```

Comment:
You're right, I put this in rather quickly and somehow it ended up working before but I just retested it. I'll need to fix it.